packages/provider/README.md

```diff
@@ -7,6 +7,14 @@ The @rarimo/provider TypeScript package which wraps managing multiple (EVM and n
 ![checks](https://badgen.net/github/checks/rarimo/js-sdk/main)
 
 ## Changelog
+
+## Unreleased
+#### Added
+- Support of Phantom wallet
+- Support of Coinbase wallet
+- Support of Solfare wallet
+#### Fixed
```

Comment:
refactor changes to the same way as below and move below to others
<img width="900" alt="image" src="https://user-images.githubusercontent.com/47495186/220383793-9028db61-94ac-4989-ba56-6018b9b5a222.png">

Reply:
fixed

Reply:
fixed